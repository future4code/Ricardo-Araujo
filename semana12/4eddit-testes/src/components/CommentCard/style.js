import styled from 'styled-components';
import ArrowUpwardIcon from '@material-ui/icons/ArrowUpward';
import ArrowDownwardIcon from '@material-ui/icons/ArrowDownward';
import Typography from '@material-ui/core/Typography';
import Paper from '@material-ui/core/Paper'

export const CommentCardWrapper = styled.div `

`

export const VotesCommentWrapper = styled.div `
    display: flex;
    justify-content: center;
    gap: 8px;
`

export const ArrowThumbsUp = styled(ArrowUpwardIcon) `
    cursor: pointer;
    color: darkgreen;
`

export const ArrowThumbsDown = styled(ArrowDownwardIcon) `
    cursor: pointer;
    color: darkred;
`

export const TypographyUser = styled(Typography) `
    
`
export const TypographyComment = styled(Typography) `

`
export const TypographyVotes = styled(Typography) `

`
export const TypographyVoteDirection = styled(Typography) `

`
export const PaperCommentCard = styled(Paper) `
    display: flex;
    flex-direction: column;
    justify-items: center;
    width: 300px;
    margin: 0 auto;
    margin-top: 20px;
    padding: 10px;
`