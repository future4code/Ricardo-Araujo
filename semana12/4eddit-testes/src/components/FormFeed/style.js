import styled from "styled-components";
import { Typography, Button } from "@material-ui/core";


export const CreatePostWrapper = styled.div`
    display: flex;
    justify-content: center;
`

export const FormFeedPage = styled.form `
display: flex;
flex-direction: column;
gap: 8px;
`

export const TypographyFeedPage = styled(Typography) `
text-align: center;
margin-top: 10px;
`

export const ButtonFeedPage = styled(Button) `
color: #fff;
font-weight: bold;
`