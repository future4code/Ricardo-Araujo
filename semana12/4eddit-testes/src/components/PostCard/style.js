import styled from 'styled-components';
import Paper from '@material-ui/core/Paper'
import Typography from '@material-ui/core/Typography'
import Button from '@material-ui/core/Button'
import ArrowUpwardIcon from '@material-ui/icons/ArrowUpward';
import ArrowDownwardIcon from '@material-ui/icons/ArrowDownward';

export const PostCardWrapper = styled.div `
    height: 100%;
    width: 100%;
    display: flex;
    flex-direction: column;
    justify-items: center;
    align-items: center;
`

export const PaperPostCard = styled(Paper) `
    display: flex;
    flex-direction: column;
    justify-items: center;
    width: 300px;
    margin: 0 auto;
    margin-top: 50px;
`

export const TypographyPostCard = styled(Typography) `
    text-align: center;
    margin-bottom: 30px;    
`

export const TypographyUser = styled(Typography) `
    font-style: italic;
    color: lightgray;
`

export const ButtonPostCard = styled(Button) `
    color: #fff;
    font-weight: bold;
`

// Votos

export const ArrowThumbsUp = styled(ArrowUpwardIcon) `
    cursor: pointer;
    color: darkgreen;
`

export const ArrowThumbsDown = styled(ArrowDownwardIcon) `
    cursor: pointer;
    color: darkred;
`

export const VotesWrapper = styled.div `
    display: flex;
    justify-content: center;
    gap: 8px;
`