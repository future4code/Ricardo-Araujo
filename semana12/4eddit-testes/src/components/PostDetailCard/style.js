import styled from 'styled-components';
import Typography from '@material-ui/core/Typography';

export const PostDetailCardWrapper = styled.div `

`

export const TypographyTitle = styled(Typography) `
    text-align: center;
`

export const TypographyUser = styled(Typography) `
    font-style: italic;
    color: lightgray;
`

export const TypographyPost = styled(Typography) `
    text-align: center;
`