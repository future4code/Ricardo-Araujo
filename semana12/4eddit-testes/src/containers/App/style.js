import styled from "styled-components"

export const AppWrapper = styled.div`
    display: flex;
    justify-content:center;
    align-items:center;
    height: 100vh;
    width: 100vw;
`

export const ContentWrapper = styled.div`
    display: grid;
    grid-template-rows: 60px 607px;
    height: 667px;
    width: 375px;
    border: 1px solid black;
`

export const RouterWrapper = styled.div`
    height: 100%;
    width: 100%;
    background-color: #f7f7f7;
    overflow-y: scroll;
`