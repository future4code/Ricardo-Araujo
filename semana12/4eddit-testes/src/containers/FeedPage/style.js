import styled from "styled-components";

export const FeedPageWrapper = styled.div`
    width:100%;
    display:flex;
    flex-direction:column;
    justify-content: center;
`
