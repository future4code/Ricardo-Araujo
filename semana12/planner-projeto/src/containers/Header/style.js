import styled from "styled-components";

export const HeaderWrapper = styled.div`
    width: 100vw;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
`