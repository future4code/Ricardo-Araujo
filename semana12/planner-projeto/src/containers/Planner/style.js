import styled from "styled-components";

export const PlannerWrapper = styled.div`
    height: 100vh;
    width: 100vw;
    background-color: orange;
    display: grid;
    grid-template-rows: 20% 1fr;
    justify-content: center;
`

export const GridWrapper = styled.div`
    grid-row:2;
    display: grid;
    grid-template-columns: repeat(7, 1fr);
    width: 90vw;
    margin: 0 auto;
`

export const DailyWrapper = styled.div`
    border: 1px solid white;
    text-align: center;
`