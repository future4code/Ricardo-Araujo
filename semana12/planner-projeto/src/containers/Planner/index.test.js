import React from 'react';
import renderer from 'react-test-renderer';
import {GridWrapper, PlannerWrapper} from "./style";

describe("Test Planner", ()=>{
    test("Snapshot do grid",()=>{
    //preparacao e execucao
    const newSnapshot = renderer.create(<GridWrapper/>).toJSON()

    //verificacao
    expect(newSnapshot).toMatchSnapshot();
    });
    test("Snapshot diagramação da tela", ()=>{
        //preparacao e execucao
        const newSnapshot = renderer.create(<PlannerWrapper/>).toJSON()

        //verificacao
        expect(newSnapshot).toMatchSnapshot();
    })
})