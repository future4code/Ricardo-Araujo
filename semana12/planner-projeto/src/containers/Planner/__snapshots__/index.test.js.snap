// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Planner Snapshot diagramação da tela 1`] = `
<div
  className="sc-bdVaJa gUXOej"
/>
`;

exports[`Test Planner Snapshot do grid 1`] = `
<div
  className="sc-bwzfXH jcuURf"
/>
`;
